superset-frontend/src/dashboard/components/menu/PopoverDropdown.tsx

```diff
@@ -98,8 +104,7 @@ class PopoverDropdown extends React.PureComponent {
     const selected = options.find(opt => opt.value === value);
     return (
       <Dropdown
-        id={id}
```

Comment:
was removing this intentional?

Reply:
Yes, because Dropdown does not have such props.